js/main.js

```diff
@@ -137,6 +143,28 @@ function viewSwap(view) {
   data.view = view;
 }
 
+$deleteButtonElement.addEventListener('click', function (event) {
+  $modalContainer.classList.remove('hidden');
+});
+
+$cancelButton.addEventListener('click', function (event) {
+  $modalContainer.classList.add('hidden');
+});
+
+$confirmButton.addEventListener('click', function (event) {
+  $modalContainer.classList.add('hidden');
+  viewSwap('entries');
+  for (let i = 0; i < data.entries.length; i++) {
+    if (data.editing === data.entries[i]) {
+      data.entries.splice(i, 1);
+      data.nextEntryId--;
```

Comment:
You shouldn't be decreasing `nextEntryId`, you only want this value to stay the same, or to increase